web/gds-user-ui/cypress/support/commands.ts

```diff
@@ -0,0 +1,21 @@
+/// <reference types="cypress" />
+// ***********************************************
+
+import '@testing-library/cypress/add-commands';
+
+// loginWith is a command that may be used to log in a user with a given email and password.
+Cypress.Commands.add('loginWith', ({ email, password }) =>
```

Comment:
I can update this to use fixtures and then add to the comments about that. I wasn't sure what would be preferred.